src/rootcheck/rootcheck-config.c

```diff
@@ -58,8 +58,9 @@ int Read_Rootcheck_Config(const char *cfgfile)
     const char *(xml_rootkit_winmalware[]) = {xml_rootcheck, "windows_malware", NULL};
     const char *(xml_scanall[]) = {xml_rootcheck, "scanall", NULL};
     const char *(xml_readall[]) = {xml_rootcheck, "readall", NULL};
+#ifdef OSSECHIDS
```

Comment:
I'm not sure if anyone uses it either. Maybe a quick post to the non dev mailing list asking if anyone uses it? Depending on the response we go from there.

Seems like a pretty safe route to go. Gives people the opportunity to save it or gives reasons as to what they use it for, etc.